performance/preparation/create-async.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+set -e
+
+# Host to use. Needs to include the protocol.
+host=$1
+# Credentials to use for the test. USER:PASS format.
+credentials=$2
+# Name of the action to create and test.
+action=$3
+
+# create a noop action
+echo "Creating action $action"
+curl -k -u "$credentials" "$host/api/v1/namespaces/_/actions/$action" -XPUT -d '{"namespace":"_","name":"test","exec":{"kind":"nodejs:default","code":"function main(){ return new Promise(function(resolve, reject) {     setTimeout(function() {       resolve({ done: true });     }, 175);   }) }"}}' -H "Content-Type: application/json"
```

Comment:
Should we rather parametrize the existing `create.sh` on the action-code and `cat` that from a file?